.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### Current Behavior
+
+- What code are you running and what is happening?
+- Include a screenshot if it makes sense.
+
+### Expected Behavior
+
+- What do you expect should be happening?
+- Include a screenshot if it makes sense.
+
+### How to reproduce
+
+- You must provide a way to reproduce the problem. If you are having an issue with your machine or build tools, the issue belongs on another repoistory as that is outside of the scope of this project.
+- Explain how to run the example and any steps that we need to take to reproduce the issue from the example app.
+
+### Your Environment
+Let us know in what environment you're running into the issue
```

Comment:
add an empty line after the header, and make the single points below bullet points, and add proper capitalization.

Reply:
done